packages/web-components/fast-ssr/src/element-renderer/elemenent-renderer.spec.ts

```diff
@@ -118,6 +118,47 @@ test.describe("FASTElementRenderer", () => {
                 <bare-element  attr><template shadowroot=\"open\"></template></bare-element>
             `);
         });
+        test("should render an attribute with a boolean value for non-aria attributes", () => {
+            const { templateRenderer } = fastSSR();
+            const result = consolidate(templateRenderer.render(html`
+                <bare-element disabled="${x => true}"></bare-element>
```

Comment:
I think this test is going to fail now too, since this will result in `disabled="true"` and `disabled="false"`

Reply:
Fix by removing this case because we don't have special "aria-*" case now